certify-sunbird-farmer.properties

```diff
@@ -0,0 +1,165 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+## ------------------------------------------- Plugin enable properties ------------------------------------------------------------
+mosip.certify.integration.scan-base-package=io.mosip.certify.sunbirdrc.integration
+mosip.certify.integration.vci-plugin=SunbirdRCVCIssuancePlugin
+mosip.certify.integration.audit-plugin=LoggerAuditService
+
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+mosip.certify.vciplugin.sunbird-rc.enable-psut-based-registry-search=false
+mosip.certify.vciplugin.sunbird-rc.issue-credential-url=${mosip.sunbird.url}/credentials-service/credentials/issue
+mosip.certify.vciplugin.sunbird-rc.supported-credential-types=FarmerCredential

```

Comment:
Shouldn't it be "FarmerProfileCredential" as per confluence
https://mosip.atlassian.net/wiki/spaces/Inji/pages/1426784342/Farmer+Credential+-+Use+Case